front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/index.tsx

```diff
@@ -71,8 +69,12 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
     Resource hooks
     ==============
   */
+  const { mutate: deleteProjectFolderFile } = useDeleteProjectFolderFile();
+  const { mutate: addProjectFolderFile } = useAddProjectFolderFile();
```

Comment:
We should use mutateAsync for both because they are added to a promise array that we wait for.